akka-docs/src/main/paradox/general/message-delivery-reliability.md

```diff
@@ -35,7 +35,7 @@ role of the “Dead Letter Office”.
 
 ## The General Rules
 
-These are the rules for message sends (i.e. the `tell` or `!` method, which
+These are the rules for message sends (i.e. the @scala[@scaladoc[tell](akka.actor.ActorRef#tell(msg:Any,sender:akka.actor.ActorRef):Unit) or @scaladoc[!](akka.actor.ActorRef#!(message:Any)(implicitsender:akka.actor.ActorRef):Unit)]@java[@javadoc[tell](akka.actor.ActorRef#tell(java.lang.Object,akka.actor.ActorRef))] method, which
 also underlies the `ask` pattern):
```

Comment:
Should I add the link for `ask`? Which link?

Reply:
Would be nice. I think https://doc.akka.io/api/akka/current/akka/pattern/AskSupport.html#ask(actorRef:akka.actor.ActorRef):akka.pattern.AskableActorRef or perhaps https://doc.akka.io/api/akka/current/akka/actor/typed/scaladsl/ActorContext.html#ask[Req,Res](target:akka.actor.typed.RecipientRef[Req],createRequest:akka.actor.typed.ActorRef[Res]=%3EReq)(mapResponse:scala.util.Try[Res]=%3ET)(implicitresponseTimeout:akka.util.Timeout,implicitclassTag:scala.reflect.ClassTag[Res]):Unit